### PR TITLE
Mm upload timeout

### PR DESCRIPTION
**Pull request by mXaln**

### CODE REVIEW

File: translationRecorder/app/src/main/java/org/wycliffeassociates/translationrecorder/FilesPage/FeedbackDialog.java [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+import android.view.View;
+import android.widget.Button;
+import android.widget.TextView;
+
+import org.wycliffeassociates.translationrecorder.R;
+import com.door43.tools.reporting.Logger;
+
+import java.io.File;
+
+public class FeedbackDialog extends DialogFragment implements View.OnClickListener {
```
Comment (jsarabia) [inline translationRecorder/app/src/main/java/org/wycliffeassociates/translationrecorder/FilesPage/FeedbackDialog.java:12]: rotating the screen will likely kill this dialog. I don't think this would have an impact on app stability, but is it acceptable behavior?

Comment (mXaln) [reply translationRecorder/app/src/main/java/org/wycliffeassociates/translationrecorder/FilesPage/FeedbackDialog.java:12]: Good point @sarabiaj , I didn't think of it. I'll take a look.

Comment (mXaln) [reply translationRecorder/app/src/main/java/org/wycliffeassociates/translationrecorder/FilesPage/FeedbackDialog.java:12]: @sarabiaj I checked, it doesn't crash, only message gets empty, but progress dialog crashes, when screen is  locked and unlocked again, but on screen rotation everything is ok.

File: translationRecorder/app/src/main/java/org/wycliffeassociates/translationrecorder/TranslationRecorderApp.java [hunk 2]
```diff
@@ -40,19 +40,18 @@ public void onCreate() {
...
-    private OkHttpClient getOkHttpClient() throws CertificateException, NoSuchAlgorithmException, KeyStoreException, KeyManagementException, IOException {
+    private OkHttpClient getOkHttpClient() {
         return new OkHttpClient.Builder()
                 .followRedirects(true)
                 .followSslRedirects(true)
                 .retryOnConnectionFailure(true)
-                .connectTimeout(15, TimeUnit.SECONDS)
-                .writeTimeout(30, TimeUnit.SECONDS)
+                .connectTimeout(30, TimeUnit.SECONDS)
+                .writeTimeout(0, TimeUnit.SECONDS)
```
Comment (jsarabia) [inline translationRecorder/app/src/main/java/org/wycliffeassociates/translationrecorder/TranslationRecorderApp.java]: any reason why the write timeout is 0?

Comment (mXaln) [reply translationRecorder/app/src/main/java/org/wycliffeassociates/translationrecorder/TranslationRecorderApp.java]: @sarabiaj when it's 0. it means unlimited, because we don't know what kind of connection a user will have, or we can set a very long value (an hour?)

File: translationRecorder/app/src/main/assets/chunks/ot/ecc/chunks.json [hunk 3]
```diff
@@ -1 +1 @@
...
...: "09-17", "lastvs": "18"}, {"firstvs": "1", "id": "10-01", "lastvs": "3"}, {"firstvs": "4", "id": "10-04", "lastvs": "4"}, {"firstvs": "5", "id": "10-05", "lastvs": "7"}, {"firstvs": "8", "id": "10-08", "lastvs": "9"}, {"firstvs": "10", "id": "10-10", "lastvs": "11"}, {"firstvs": "12", "id": "10-12", "lastvs": "12"}, {"firstvs": "13", "id": "10-13", "lastvs": "14"}, {"firstvs": "15", "id": "10-15", "lastvs": "15"}, {"firstvs": "16", "id": "10-16", "lastvs": "17"}, {"firstvs": "18", "id": "10-18", "lastvs": "19"}, {"firstvs": "20", "id": "10-20", "lastvs": "20"}, {"firstvs": "1", "id": "11-01", "lastvs": "3"}, {"firstvs": "4", "id": "11-04", "lastvs": "5"}, {"firstvs": "6", "id": "11-06", "lastvs": "8"}, {"firstvs": "9", "id": "11-09", "lastvs": "10"}, {"firstvs": "1", "id": "12-01", "lastvs": "2"}, {"firstvs": "3", "id": "12-03", "lastvs": "3"}, {"firstvs": "4", "id": "12-04", "lastvs": "4"}, {"firstvs": "5", "id": "12-05", "lastvs": "5"}, {"firstvs": "6", "id": "12-06", "lastvs": "7"}, {"firstvs": "8", "id": "12-08", "lastvs": "9"}, {"firstvs": "10", "id": "12-10", "lastvs": "11"}, {"firstvs": "12", "id": "12-12", "lastvs": "12"}, {"firstvs": "13", "id": "12-00", "lastvs": "14"}]
```
Comment (jsarabia) [inline translationRecorder/app/src/main/assets/chunks/ot/ecc/chunks.json:1]: why is this changed while the rest are removed?

Comment (mXaln) [reply translationRecorder/app/src/main/assets/chunks/ot/ecc/chunks.json:1]: @sarabiaj these are updated from unfoldingword api, by running python scripts included into the project. The rest are deleted, because they were just duplicates

File: translationRecorder/app/src/main/java/org/wycliffeassociates/translationrecorder/TranslationRecorderApp.java (hunk 2 above)
Comment (jsarabia) [reply translationRecorder/app/src/main/java/org/wycliffeassociates/translationrecorder/TranslationRecorderApp.java]: oh okay, cool. can you comment "//unlimited timeout"?

File: translationRecorder/app/src/main/java/org/wycliffeassociates/translationrecorder/FilesPage/FeedbackDialog.java (hunk 1 above)
Comment (jsarabia) [reply translationRecorder/app/src/main/java/org/wycliffeassociates/translationrecorder/FilesPage/FeedbackDialog.java:12]: when you say "progress dialog crashes" do you mean that it just goes away, or the app crashes?

Comment (mXaln) [reply translationRecorder/app/src/main/java/org/wycliffeassociates/translationrecorder/FilesPage/FeedbackDialog.java:12]: sorry, yes the app crashes

**Review by jsarabia — APPROVED** [review]
